worker/dbaccessor/package_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2014 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package dbaccessor_test
```

Comment:
```suggestion
package dbaccessor
```

It might be a good idea to drop the habit of having `_test` packages and collocate unit tests with the subject under test.